From: Marco Felsch <m.felsch@pengutronix.de>
Date: Wed, 27 Mar 2019 12:38:53 +0100
Subject: [PATCH] ARM: dts: imx6qdl-skov-revc: fix backlight enable-gpio
 compile warning

The commit fixes the following dtc warning:

arch/arm/boot/dts/imx6dl-skov-revc-lt2.dtb: Warning (gpios_property):
/soc/aips-bus@2000000/gpio@20b0000/backlight: Missing property
'#gpio-cells' in node /soc/aips-bus@2000000/iomuxc@20e0000/skov_imx6_grp/ads7846grp
or bad phandle (referred from gpios[0])

The gpio-initval property was discussed here:
https://www.spinics.net/lists/devicetree/msg105438.html. It seems that
it didn't landed into mainline. Anyway the backlight is off till someone
enable it so we can remove it.

Signed-off-by: Marco Felsch <m.felsch@pengutronix.de>
---
 arch/arm/boot/dts/imx6qdl-skov-cpu-revc.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-skov-cpu-revc.dtsi b/arch/arm/boot/dts/imx6qdl-skov-cpu-revc.dtsi
index 97284a974c1c..2338e2cc7347 100644
--- a/arch/arm/boot/dts/imx6qdl-skov-cpu-revc.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-skov-cpu-revc.dtsi
@@ -45,14 +45,6 @@
 	enable-gpios = <&gpio6 23 GPIO_ACTIVE_LOW>;
 };
 
-&gpio6 {
-	backlight {
-		gpios = <23 0>; /* initially switch off the panel backlight */
-		output-high;
-		gpio-initval;
-	};
-};
-
 &ads7846 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_ads7846>;
